### PR TITLE
Implement CI pipeline

### LINKED ISSUE #10: Add CI
currently the docs do not get automatically deployed.

### PR DESCRIPTION
**Pull request by sameersbn**
Closes #10

### CODE REVIEW

File: scripts/release-notes.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
```
Comment (antoineco) [inline scripts/release-notes.sh:1]: Do we need this script at all?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,9 @@
-FROM nginx
+FROM gcr.io/triggermesh/mkdocs:latest AS builder
```
Comment (antoineco) [inline Dockerfile:1]: Where is the source of that image, and what's the rational for not using the `python` image?

File: scripts/release-notes.sh (hunk 1 above)
Comment (sameersbn) [reply scripts/release-notes.sh:1]: i am using it for generating the release notes in the ci job.

File: Dockerfile (hunk 2 above)
Comment (sameersbn) [reply Dockerfile:1]: note sure where the source lives, but you're right we can just replace it with plain old python image.

File: scripts/release-notes.sh (hunk 1 above)
Comment (antoineco) [reply scripts/release-notes.sh:1]: What I mean is: why having release notes for something we update and release continuously?

Comment (sameersbn) [reply scripts/release-notes.sh:1]: well with this CI in place the docs will (now) be updated at the same time as the monthly release.

File: Dockerfile (hunk 2 above)
Comment (sameersbn) [reply Dockerfile]: updated!

**Review by antoineco — APPROVED** [review]

**Pull request merged**